libs/user-identity-service/src/test/java/uk/gov/di/ipv/core/library/service/UserIdentityServiceTest.java

```diff
@@ -733,15 +779,50 @@ void shouldNotSetIdentityClaimWhenVotIsP0() throws Exception {
         when(mockDataStore.getItems(anyString())).thenReturn(vcStoreItems);
         mockParamStoreCalls(paramsToMockForP0WithNoCi);
 
+        // Act
         UserIdentity credentials =
                 userIdentityService.generateUserIdentity(
                         USER_ID_1, "test-sub", "P0", emptyContraIndicators);
 
+        // Assert
         assertNull(credentials.getIdentityClaim());
     }
 
+    @Test
```

Comment:
Here are the tests I added for the `getVot` method I introduced for the `UserIdentityService`